### PR TITLE
Debian support on RMT

### PR DESCRIPTION
**Pull request by ngetahun**
## Description

Please describe your change and provide as much context as possible.

Fixes # (issue)

## Change Type

*Please select the correct option.*

- [ ] **Bug Fix** (a non-breaking change which fixes an issue)
- [ ] **New Feature** (a non-breaking change which adds new functionality)
- [ ] **Documentation Update** (a change which only updates documentation)

## Checklist

*Please check off each item if the requirement is met.*

- [ ] I have verified that my code follows RMT's coding standards with `rubocop`.
- [ ] I have reviewed my own code and believe that it's ready for an external review.
- [ ] I have provided comments for any hard-to-understand code.
- [ ] I have documented the `MANUAL.md` file with any changes to the user experience.
- [ ] RMT's test coverage remains at 100%.
- [ ] If my changes are non-trivial, I have added a changelog entry to notify users at `package/obs/rmt-server.changes`.

## Other Notes

Please use this space to provide notes or thoughts to the team, such as tips on how to review/demo your changes.

### CODE REVIEW

**Comment by suse-tests-fail** [comment]
Prophet reports failure.
https://ci.suse.de/job/scc-RMT-integration-tests/355893/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
Failed fast in project rmt on step docker-compose-build-images. (Merged 06ce2678b673b3aaa61b3c79ad7f885a4e9e6cef into ecf5efb3d9dbfce20f8d302a5ae4db03e08f611d)

**Comment by suse-tests-fail** [comment]
Prophet reports failure.
https://ci.suse.de/job/scc-RMT-integration-tests/355922/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
Failed fast in project rmt on step docker-compose-build-images. (Merged ff79005ff1e8edb4a8e12f77986e9fc8e4d652a7 into ecf5efb3d9dbfce20f8d302a5ae4db03e08f611d)

**Comment by suse-tests-fail** [comment]
Prophet reports failure.
https://ci.suse.de/job/scc-RMT-integration-tests/356056/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
Failed fast in project rmt on step docker-compose-build-images.
Failed fast in project rmt on step docker-compose-build-images. (Merged 0aa6dc9a6eeca929ba95f4992879c50abee671f8 into ecf5efb3d9dbfce20f8d302a5ae4db03e08f611d)

**Comment by suse-tests-fail** [comment]
Prophet reports failure.
https://ci.suse.de/job/scc-RMT-integration-tests/356667/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
Failed fast in project rmt on step docker-compose-build-images. (Merged 0aa6dc9a6eeca929ba95f4992879c50abee671f8 into ecf5efb3d9dbfce20f8d302a5ae4db03e08f611d)

File: package/obs/rmt.conf [hunk 1]
```diff
@@ -7,6 +7,14 @@ database:
   encoding: utf8
   timeout: 5000
   pool: 5
+  # set sslverify to true to use tls over ssl mysql connection
+  sslverify: false
+  sslkey: /some/path
+  sslcert: /some/path
+  sslca: /some/path
```
Comment (ngetahun) [inline package/obs/rmt.conf:14]: Do we know why/when this was added?

Comment (felixsch) [reply package/obs/rmt.conf:14]: This is a remnant of testing you did about the mysql connection. We should drop it!

**Review by felixsch — APPROVED** [review]
lgtm :+1:

**How I tested this pullrequest:**

- Ran `rmt-cli mirror` with:
  - Tested mirroring bz2 compressed repomd repositories
  - Tested regular repomd repositories
  - Tested custom repositories
  - Tested mirroring SUMA Tools debian repository
 
**As always, if you think I missed something please let me know!**:rocket:

**Pull request merged**